pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -83,8 +83,8 @@ stdenv.mkDerivation rec {
     "-Dldconfig=false"
     "-Dsmack=true"
     "-Db_pie=true"
-    "-Dsystem-uid-max=499" #TODO: debug why awking around in /etc/login.defs doesn't work
-    "-Dsystem-gid-max=499"
+    "-Dsystem-uid-max=999" #TODO: debug why awking around in /etc/login.defs doesn't work
```

Comment:
Do you understand what this `TODO` is about?

Reply:
I kind of took it as "debug why login.defs settings aren't used", but I am a bit confused about the "awking", what does `awk` have to do with this??

Reply:
Yeah, I also think it’s referring to that. In that case, should we replace the `TODO` with a link to the systemd issue?

Reply:
Agreed. Apparently there's a couple issues on that: https://github.com/systemd/systemd/issues/3855, https://github.com/systemd/systemd/issues/4850, https://github.com/systemd/systemd/issues/9769, https://github.com/systemd/systemd/issues/9843, https://github.com/systemd/systemd/issues/10184

Guess I'll link to https://github.com/systemd/systemd/issues/9769